cmd/terramate/cli/cli.go

```diff
@@ -109,144 +109,140 @@ type kongParallelFlag struct {
 const defaultParallelRunCount = 5
 
 type cliSpec struct {
-	Version        struct{} `cmd:"" help:"Terramate version"`
-	VersionFlag    bool     `name:"version" help:"Terramate version"`
-	Chdir          string   `short:"C" optional:"true" predictor:"file" help:"Sets working directory"`
-	GitChangeBase  string   `short:"B" optional:"true" help:"Git base ref for computing changes"`
-	Changed        bool     `short:"c" optional:"true" help:"Filter by changed infrastructure"`
-	Tags           []string `optional:"true" sep:"none" help:"Filter stacks by tags. Use \":\" for logical AND and \",\" for logical OR. Example: --tags app:prod filters stacks containing tag \"app\" AND \"prod\". If multiple --tags are provided, an OR expression is created. Example: \"--tags a --tags b\" is the same as \"--tags a,b\""`
-	NoTags         []string `optional:"true" sep:"," help:"Filter stacks that do not have the given tags"`
-	LogLevel       string   `optional:"true" default:"warn" enum:"disabled,trace,debug,info,warn,error,fatal" help:"Log level to use: 'disabled', 'trace', 'debug', 'info', 'warn', 'error', or 'fatal'"`
-	LogFmt         string   `optional:"true" default:"console" enum:"console,text,json" help:"Log format to use: 'console', 'text', or 'json'"`
-	LogDestination string   `optional:"true" default:"stderr" enum:"stderr,stdout" help:"Destination of log messages"`
-	Quiet          bool     `optional:"false" help:"Disable output"`
+	VersionFlag    bool     `hidden:"true" name:"version" help:"Show Terramate version."`
+	Chdir          string   `short:"C" optional:"true" predictor:"file" help:"Set working directory."`
+	GitChangeBase  string   `short:"B" optional:"true" help:"Set git base reference for computing changes."`
+	Changed        bool     `short:"c" optional:"true" help:"Filter stacks based on changes made in git."`
+	Tags           []string `optional:"true" sep:"none" help:"Filter stacks by tags."`
+	NoTags         []string `optional:"true" sep:"," help:"Filter stacks by tags not being set."`
+	LogLevel       string   `optional:"true" default:"warn" enum:"disabled,trace,debug,info,warn,error,fatal" help:"Log level to use: 'disabled', 'trace', 'debug', 'info', 'warn', 'error', or 'fatal'."`
+	LogFmt         string   `optional:"true" default:"console" enum:"console,text,json" help:"Log format to use: 'console', 'text', or 'json'."`
+	LogDestination string   `optional:"true" default:"stderr" enum:"stderr,stdout" help:"Destination channel of log messages: 'stderr' or 'stdout'."`
+	Quiet          bool     `optional:"false" help:"Disable outputs."`
```

Comment:
The `--quiet` is at the moment behaving as "disable helper messages", as it only disables the messages like "Entering directory...", "Executing ...", etc, but the actual output of the command is still printing.

This flag is heavily used in the e2e tests, so we hide all non-important messages and just assert for the important output but the `--help` for it is a bit misleading, as it is disabling all outputs.

Reply:
this is fine. we did not say we disable outputs of the commands executed.